vault/logical_system.go

```diff
@@ -4079,7 +4081,11 @@ func (b *SystemBackend) pathInternalOpenAPI(ctx context.Context, req *logical.Re
 					}
 				}
 
-				doc.Paths["/"+mountPrefix+mount+path] = obj
+				if genericPaths && mount != "sys/" && mount != "identity/" {
+					doc.Paths["/"+mountPrefix+"{mountPath}/"+path] = obj
```

Comment:
I had to stare at this for a few minutes 😆 so just to make sure I'm following - `{mountPath}` is a string literal here, yes? Not some kind of variable interpolation I'm not familiar with?

Reply:
Haha. Yes! it is just a string literal here. That's how we want it to look in the spec,  but I can add a comment for clarification.

Reply:
Using `fmt.Sprintf` to construct the string might make it more readable